packages/next-server/server/next-server.ts

```diff
@@ -266,15 +267,27 @@ export default class Server {
       return this.render404(req, res, parsedUrl)
     }
 
-    const html = await this.renderToHTML(req, res, pathname, query, {
-      dataOnly: this.renderOpts.ampBindInitData && Boolean(query.dataOnly) || (req.headers && (req.headers.accept || '').indexOf('application/amp.bind+json') !== -1),
-    })
-    // Request was ended by the user
-    if (html === null) {
-      return
-    }
+    try {
+      const html = await this.renderToHTML(req, res, pathname, query, {
+        handleError: false,
+        dataOnly: this.renderOpts.ampBindInitData && Boolean(query.dataOnly) || (req.headers && (req.headers.accept || '').indexOf('application/amp.bind+json') !== -1),
+      })
+      // Request was ended by the user
+      if (html === null) {
+        return
+      }
 
-    return this.sendHTML(req, res, html)
+      return this.sendHTML(req, res, html)
+    } catch (err) {
+      if (err && err.code === 'ENOENT') {
```

Comment:
This isn't reached in dev mode, will need to add a check in `next-dev-server` `renderToHTML`. Should probably add a test to make sure it's working in dev mode

Reply:
@ijjk Done, thank you.